bootloader/src/pyi_launch.c

```diff
@@ -467,10 +469,12 @@ int pyi_launch_execute(ARCHIVE_STATUS *status)
     if (!*PI_Py_FileSystemDefaultEncoding) {
         char *saved_locale, *loc_codeset;
         saved_locale = strdup(setlocale(LC_CTYPE, NULL));
```

Comment:
Thanks for the explanation.

(BTW: I doubt, that "C" is the usual locale on modern Linux systems. I assume it to be a country-specific locale in most of the times and some UTF-8 locale in other cases.)